python/sglang/srt/layers/moe/topk.py

```diff
@@ -222,6 +235,7 @@ def select_experts(
                 renormalize=renormalize,
                 num_expert_group=num_expert_group,
                 topk_group=topk_group,
+                share_fusion=int(os.getenv("SHARE_EXPERTS_FUSION_REPLICA", "0")) > 0,
```

Comment:
nit: it seems the env var is used several times, thus maybe we can extract it to a global variable (constant)